MELA/src/TEvtProb.cc

```diff
@@ -207,11 +207,18 @@ void TEvtProb::SetVerbosity(TVar::VerbosityLevel tmp){ verbosity = tmp; }
 void TEvtProb::SetLeptonInterf(TVar::LeptonInterference tmp){ leptonInterf = tmp; }
 void TEvtProb::SetCandidateDecayMode(TVar::CandidateDecayMode mode){ PDGHelpers::setCandidateDecayMode(mode); }
 void TEvtProb::SetRenFacScaleMode(TVar::EventScaleScheme renormalizationSch, TVar::EventScaleScheme factorizationSch, double ren_sf, double fac_sf){
+  if(verbosity>=TVar::DEBUG_VERBOSE){
+    MELAout << "Renormalization Scheme/Scale Factor set to " << renormalizationSch << "/" << ren_sf << endl;
+    MELAout << "Factorization Scheme/Scale Factor set to " << factorizationSch << "/" << fac_sf << endl;
+  } 
```

Comment:
I would either remove these or remake the reference files in case any of them call SetRenFacScaleMode.